pyuvdata/uvdata/tests/test_uvfits.py

```diff
@@ -779,9 +848,32 @@ def test_multi_files(casa_uvfits, tmp_path):
     )
 
     uv1.history = uv_full.history
+
+    # make sure filenames are what we expect
+    assert set(uv1.filename) == {"uv1.uvfits", "uv2.uvfits"}
+    assert uv_full.filename == ["day2_TDEM0003_10s_norx_1src_1spw.uvfits"]
+    uv1.filename = uv_full.filename
+    uv1._filename.form == (1,)
+
     assert uv1 == uv_full
 
-    # again, setting axis
+
+@pytest.mark.filterwarnings("ignore:Telescope EVLA is not")
+@pytest.mark.filterwarnings("ignore:The uvw_array does not match the expected values")
+def test_multi_files_axis(casa_uvfits, tmp_path):
+    """
+    Reading multiple files at once using "axis" keyword.
+    """
+    uv_full = casa_uvfits
+    testfile1 = str(tmp_path / "uv1.uvfits")
```

Comment:
I'm really interested in your aversion to using the `/` operator on pathlib.Path objects (specifically this object and subclasses thereof). The python documentation even states they behave similariy.

Reply:
hmm, I guess it's because I didn't realize it was handled properly on those objects. It derives from bad experiences with things not being handled properly across different operating systems. If it works and is documented as proper in the docs I withdraw my objection.